src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -8,21 +8,27 @@ description: "Learn about the Azure Active Directory Single Sign-On (SSO)."
 
 1. Log in to the [Azure portal](https://portal.azure.com/).
 
-2. (If you don't require a permission group for Sentry, skip to Step 4.) Under Manage Azure Active Directory (the shield picture), click "View", which takes you to the Organization Overview page. Now, proceed to Step 3.
+1. Under "Manage Azure Active Directory" (the shield picture), click "View", which takes you to the Organization Overview page.
```

Comment:
Changing all ordered list items to `1.` because it doesn't make sense to change the number for every line when I want to add a line somewhere in the middle

https://www.gatsbyjs.com/docs/reference/markdown-syntax/#ordered